### PR TITLE
Add option of pushing to Quay

### PR DESCRIPTION
**Pull request by querti**
If neither --skip-quay nor --skip-pulp is specified, only a push
to Quay is performed. Pubtools-quay is used for pushing to Quay.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[report]
+show_missing = True
+fail_under = 90
```
Comment (midnightercz) [inline .coveragerc:3]: Can we aim to 100?

File: README.rst [hunk 2]
```diff
@@ -51,4 +53,30 @@ Usage
...
+  $ export SSH_PASSWORD=ssh_password
+  $ pubtools-iib-add-bundles \
+    --iib-server iibhostname.example.com \
+    --binary-image container-registry.example.com/binary/image:latest \
+    --index-image container-registry.example.com/index/image:latest \
+    --bundle container-registry.example.com/bundle/image:123 \
+    --arch x86_64 \
+    --skip-pulp \
+    --quay-dest-repo quay.io/namespace/repo \
+    --quay-user namespace+robot_account \
```
Comment (midnightercz) [inline README.rst:71]: why there's namespace+robot_account? Shouldn't that be only robot_account?

File: .coveragerc (hunk 1 above)
Comment (querti) [reply .coveragerc]: The coverage was already at 100%, I changed the constraint.

File: README.rst (hunk 2 above)
Comment (querti) [reply README.rst:71]: This is the format of Quay robot accounts

**Review by midnightercz — APPROVED** [review]

**Review by emilyzheng — APPROVED** [review]

**Pull request merged**